avocado/utils/network/interfaces.py

```diff
@@ -380,3 +380,15 @@ def restore_from_backup(self):
         else:
             raise NWException(
                 "Backup file not available, could not restore file.")
+
+    def interface_present(self):
```

Comment:
To follow other methods patterns, I would go with `is_interface_available` or `is_interface_present`.

Reply:
Makes sense!